app/gui/qt/mainwindow.h

```diff
@@ -121,6 +121,7 @@ private slots:
     bool saveDialog();
     bool shareDialog();
     void load();
```

Comment:
In face we could potentially overload the `load()` function to take an optional parameter for filename which would skip the selection process.